main.py

```diff
@@ -44,6 +48,37 @@ def signup_data():
             cur.close()
             return render_template("chef.html")
 
+@app.route("/login", methods=['POST', 'GET'])
+def login_data():
+    msg = ""
+    if request.method == 'POST' and 'name' in request.form and 'password' in request.form:
+        name = request.form['name']
```

Comment:
Are you checking name and password for the login? Or are you supposed to check email and password?

Reply:
I'm checking the name and password for the login.